src/styles/product-card.scss

```diff
@@ -0,0 +1,89 @@
+@import 'fonts';
+@import 'stars';
+@import 'variables';
```

Comment:
this is already connected in the main file - `index.scss`. u don't need to reconnect it